flocker/node/agents/gce.py

```diff
@@ -350,7 +353,7 @@ def _disk_resource_description(self):
         """
         return u"flocker-v1-cluster-id: " + unicode(self._cluster_id)
 
-    def _do_blocking_operation(self, function, **kwargs):
+    def _do_blocking_operation(self, function, max_wait=60, **kwargs):
```

Comment:
Lol, Thanks-fixed. Actually that is definitely something I would have complained about later. I sincerely appreciate you catching that before I put that into master.